react/react1/week1/todo_list/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+function TodoItem(props) {
+  return (
+    <div>
+      <h3>{props.description}</h3>
+      <p>{props.deadline}</p>
+    </div>
+  );
+}
```

Comment:
Please put components to separate files (TodoItem.js)

Reply:
Ok, sure. I'll do it now. Thank you